src/features/RequestForm/RequestVacation/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import { requestVacationTitle } from 'features/constants'
+import RequestTemplates from 'features/RequestForm/RequestTemplates'
+import listVacation from 'pages/portal/formListVacation'
+
+const RequestVacation = () => {
```

Comment:
above you have a function RequestMaterials, which is pretty much the same as this one. Make you could try making a more generic function?